src/components/ListOfGoods.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import './ListOfGoods.scss';
+
+type State = {
+  isReversed: boolean;
+  sortBy: string;
+};
+
+type Props = {
+  goods: string[];
+};
+
+export class ListOfGoods extends React.Component<Props, State> {
+  state = {
+    isReversed: false,
+    sortBy: '',
+  };
+
+  reverse = () => {
+    this.setState(currentState => ({
+      isReversed: !currentState.isReversed,
+    }));
+  };
+
+  sortByAlphabet = () => {
+    this.setState({ sortBy: 'alphabet' });
+  };
+
+  sortByLength = () => {
+    this.setState({ sortBy: 'length' });
+  };
```

Comment:
You can make only one method 
```suggestion
  sortByValue = (value: string) => {
    this.setState({ sortBy: value });
  };
```